CHANGELOG.md

```diff
@@ -20,6 +20,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `set(user:)` is not required for query (channels, users) unless `presence: true` or `state: true` is specified [#269](https://github.com/GetStream/stream-chat-swift/issues/269)
 - Fix crash in iOS12 caused by abstract URLSession instance [#272](https://github.com/GetStream/stream-chat-swift/issues/272)
 - Fix infinite loop when the web socket connection fails (iOS13 only) [#273](https://github.com/GetStream/stream-chat-swift/pull/273).
+- Use user names and avatars for channels 1-by-1 [#275](https://github.com/GetStream/stream-chat-swift/issues/275).
+ ```swift
+let anotherUser = User("second")
+anotherUser.name = "John"
+anotherUser.avatarURL = URL(string: "http://example.com/john")
+
+let channel = client.channel(members: [client.user, anotherUser])
```

Comment:
I don't want to refer to a shared client.

Reply:
It's what we have now, better to have code examples working right away. Also, this code example implies our `Client` is usable without `shared` but it's not the case, everything is built upon singleton

Reply:
What if developers use Stream Client as own StreamChatService and hold the client as a variable. For now, they have a reference to the shared instance, but they do that because they don't want to use our singleton inside own project in different places. This is as I understand in usage from some experienced developers and teams. So for them, it's just a client of their own service for internal service usage. I'm not sure we should always refer to the shared instance.